testsuite/classlibrary/TestTask.sc

```diff
@@ -23,4 +23,31 @@ TestTask : UnitTest {
 		this.assert(task.isPlaying.not, "Task be stoppable even if not running");
 	}
 
+	test_play_with_SystemClock {
+		this.makePlayTestWithClock(SystemClock, "SystemClock");
+	}
+
+	test_play_with_AppClock {
+		this.makePlayTestWithClock(AppClock, "AppClock");
+	}
+
+	test_play_with_TempoClock {
+		this.makePlayTestWithClock(TempoClock.default, "TempoClock.default");
```

Comment:
I'd have to suggest `TempoClock.new` here, because it will be `stop`-ped later. (As a rule of thumb, if this block of code didn't create the clock, then it shouldn't be stopping the clock.)